openmc/settings.py

```diff
@@ -335,6 +335,213 @@ def __init__(self, **kwargs):
+
+    @property
+    def surf_source_write(self) -> dict:
+        return self._surf_source_write
+
+    @property
+    def no_reduce(self) -> bool:
+        return self._no_reduce
+
+    @property
+    def verbosity(self) -> int:
+        return self._verbosity
+
+    @property
+    def tabular_legendre(self) -> dict:
+        return self._tabular_legendre
+
+    @property
+    def temperature(self) -> dict:
+        return self._temperature
+
+    @property
+    def trace(self) -> typing.Iterable:
+        return self._trace
+
+    @property
+    def track(self) -> typing.Iterable[typing.Iterable[int]]:
+        return self._track
+
+    @property
+    def cutoff(self) -> dict:
+        return self._cutoff
+
+    @property
+    def ufs_mesh(self) -> RegularMesh:
+        return self._ufs_mesh
+
+    @property
+    def resonance_scattering(self) -> dict:
+        return self._resonance_scattering
+
+    @property
+    def volume_calculations(self) -> typing.List[VolumeCalculation]:
+        return self._volume_calculations
+
+    @property
+    def create_fission_neutrons(self) -> bool:
+        return self._create_fission_neutrons
+
+    @property
+    def create_delayed_neutrons(self) -> bool:
+        return self._create_delayed_neutrons
+
+    @property
+    def delayed_photon_scaling(self) -> bool:
+        return self._delayed_photon_scaling
+
+    @property
+    def material_cell_offsets(self) -> bool:
+        return self._material_cell_offsets
+
+    @property
+    def log_grid_bins(self) -> int:
+        return self._log_grid_bins
+
+    @property
+    def event_based(self) -> bool:
+        return self._event_based
+
+    @property
+    def max_particles_in_flight(self) -> int:
+        return self._max_particles_in_flight
+
+    @property
+    def write_initial_source(self) -> bool:
+        return self._write_initial_source
+
+    @property
+    def weight_windows(self) -> typing.List[WeightWindows]:
+        return self._weight_windows
+
+    @property
+    def weight_windows_on(self) -> bool:
+        return self._weight_windows_on
+
+    @property
+    def weight_windows_file(self) -> Optional[PathLike]:
+        return self._weight_windows_file
+
+    @weight_windows_file.setter
+    def weight_windows_file(self, value: PathLike):
+        cv.check_type('weight windows file', value, (str, Path))
+        self._weight_windows_file = value
+
+    @property
+    def weight_window_generators(self) -> typing.List[WeightWindowGenerator]:
+        return self._weight_window_generators
+
+    @weight_window_generators.setter
+    def weight_window_generators(self, wwgs):
+        if not isinstance(wwgs, MutableSequence):
+            wwgs = [wwgs]
+        self._weight_window_generators = cv.CheckedList(WeightWindowGenerator, 'weight window generators', wwgs)
+
+    @property
+    def max_splits(self) -> int:
+        return self._max_splits
+
+    @property
+    def max_tracks(self) -> int:
+        return self._max_tracks
+
```

Comment:
Looks like you have a big block of code here that got duplicated while resolving merge conflicts

Reply:
Ugh. My bad. Thanks!